docs/source/client.rst

```diff
@@ -14,15 +14,15 @@ it to the address of a ``Scheduler``:
 There are a few different ways to interact with the cluster through the client:
 
 1.  The Client satisfies most of the standard concurrent.futures_ - PEP-3148_
-    interface with ``.submit``, ``.map`` functions and ``Future`` objects,
+    interface with ``.submit``, ``.map`` functions and ``Task`` objects,
     allowing the immediate and direct submission of tasks.
 2.  The Client registers itself as the default Dask_ scheduler, and so runs all
     dask collections like dask.array_, dask.bag_, dask.dataframe_ and dask.delayed_
 3.  The Client has additional methods for manipulating data remotely.  See the
     full :doc:`API <api>` for a thorough list.
 
 
-Concurrent.futures
```

Comment:
Again more examples of over-zealous renaming

Reply:
(totally understandable though that this would happen in a first pass.  I'm just trying to point out a few exemplars so that it's easier to find them when cleaning up.)

Reply:
Thanks for looking! First iteration was just trying to get tests happy, and ya, see there are a number of other rules to add to avoid the over-zealous renaming. Nice to have some of them pointed out. :+1: